packages/cozy-stack-client/src/mangoIndex.js

```diff
@@ -53,7 +51,7 @@ export const normalizeDesignDoc = designDoc => {
  *
  * @param {Array<string>} fields - The indexed fields
  * @param {object} params - The additional params
- * @param {Array<string>} params.partialFilterFields - The partial filter fields
+ * @param {Array<string>=} params.partialFilterFields - The partial filter fields
```

Comment:
Typo?

Reply:
No, it means "optional". https://jsdoc.app/tags-param.html

Reply:
oh, TIL. It feels kind of weird, but nice to know :+1: